Makefile

```diff
@@ -41,6 +41,11 @@ fast_tests_diffusers:
 	python -m pip install .[tests]
 	python -m pytest tests/test_diffusers.py
 
+# Run unit and integration tests related to text feature extraction
+fast_tests_feature_extraction:
```

Comment:
My suggestion would be to add this under fast_tests but will defer to @regisss on this